src/ssh_cert_service/settings.py

```
@@ -2,3 +2,9 @@
 #
 # This file is read before all other configuration sources
 #
+
+import os
+
+# get path from the masterkey is located 
+MASTER_PRIVATE_KEY_PATH = os.environ.get("COESRA_PRIVATE_KEY_PATH_ENV") 
```

If these are required to run the application, you can set it up to fail by accessing it directly using `[]` instead of `get()`. It'll throw a key error if it doesn't exist.

```
MASTER_PRIVATE_KEY_PATH = os.environ["COESRA_PRIVATE_KEY_PATH_ENV"]
```

This way it will fail early in your deployment instead of at runtime.